src/OpprettAvtale/OpprettAvtale.tsx

```diff
@@ -131,12 +143,18 @@ class OpprettAvtale extends React.Component<Context & RouterProps, State> {
                 </EkstbanderbartPanelRad>
 
                 <EkstbanderbartPanelRad svgIkon={<NokkelPunktForAvtale />}>
-                    For at avtalen skal kunne opprettes, må avtalens parter bli
-                    knyttet sammen med gyldige, unike identifikasjoner:
-                    fødselsnummer. På sikt kan alle bruke Altinn til å logge
-                    inn, men enn så lenge må partene i avtalen utveksle sitt
-                    fødselsnummer. Arbeidsgiver sitt fødselsnummer er ikke
-                    synlig i avtalen.
+                    For at deltaker og arbeidsgiver skal få tilgang til avtalen
+                    må de logge seg inn via ID-porten. Tilgang for arbeidsgiver
+                    styres gjennom Altinn. For at en arbeidsgiver kan
+                    representere en bedrift må personen ha rollen{' '}
+                    <em>Helse-, sosial- og velferdstjenester</em> eller gis
+                    tilgang til enkelttjenesten{' '}
+                    <em>Avtale om arbeidstrening</em>. Mer informasjon om roller
+                    og rettigheter finnes hos{' '}
+                    <Lenke href="https://www.altinn.no/hjelp/profil/roller-og-rettigheter/">
```

Comment:
Ønskelig å kanskje åpne lenken i en ny tab/vindu istedenfor å navigere vekk fra løsningen? isåfall target="_blank"

Reply:
Har lagt til `target="_blank"`